packages/trader/src/sass/app/modules/settings.scss

```diff
@@ -122,3 +122,12 @@
         padding: 0;
     }
 }
+
+/** @define modal-header; */
```

Comment:
Do we have to do this style definition every time we use modals?

Reply:
@reza-fs if you have to override some style for modal, specific to your component you need to. though you can also send a header component with any style you want instead of sending title and overriding it.